File: 240817/알파벳에 따른 평가/evaluation-by-alphabet.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String text = sc.next();

        if ( text.equals("S"))
            System.out.print("Superior");
        else if (text.equals("A"))
            System.out.print("Exellent");
        else if (text.equals("B"))
            System.out.print("Good");
        else if (text.equals("C"))
            System.out.print("Usually");
        else if (text.equals("D"))
            System.out.print("Effort");
        else 
            System.out.print("Failure");

    }
}

File: 250228/알파벳에 따른 평가/evaluation-by-alphabet.js
const fs = require("fs");

let arr = fs.readFileSync(0).toString().trim();

if(arr == 'S'){
    console.log("Superior");
} else if(arr == 'A'){
    console.log("Excellent");
} else if(arr == 'B'){
    console.log("Good");
} else if (arr == 'C'){
    console.log("Usually");
} else if (arr == 'D'){
    console.log("Effort");
} else {
    console.log("Failure");
}